src/NuGet.Services.EndToEnd/Support/Package.cs

```diff
@@ -44,18 +44,19 @@ public override string ToString()
             return $"{Id} {FullVersion}";
         }
 
-        public static Package Create(string id, string normalizedVersion)
+        public static Package Create(string id, string normalizedVersion, PackageType packageType)
```

Comment:
>Create [](start = 30, length = 6)

Nit: `PackageType` should be the first parameter since it is the most important in the scope of E2E tests. It acts like the identifier. You can use Visual Studio refactoring tools to rearrange parameters easily.

Reply:
Done!